### ISSUE TITLE
Clicking Ads in the notification center on macOS do not open the Ad

### ISSUE THREAD

**Issue by tmancey** [brave/brave-browser#2610]
## Description 
Clicking Ads in the notification center do not open the Ad

## Steps to Reproduce

   1.  Visit a website, e.g. www.foodnetwork.com
   2. Remain idle for 15 seconds
   3. Move the mouse to become unidle
   4. Ad is shown
   5. Open Notification Center
   6. Click Ad in notification center

## Actual result:
Ad does not open in browser

## Expected result:
Ad should open open in browser

## Reproduces how often: 
Everytime

## Brave version (brave://version info)
macOS Version 0.59.8 (64-bit)

**Comment by bridiver** [comment]
how long should it take for the ad to disappear? Mine doesn't seem to ever go away

**Comment by tmancey** [comment]
What settings do you have in macOS Preferences/Notifications?

**Comment by tmancey** [comment]
> how long should it take for the ad to disappear? Mine doesn't seem to ever go away

@bridiver Issue resolved by Brave Ads notifications disappear without user input https://github.com/brave/brave-browser/issues/2608

**Comment by GeetaSarvadnya** [comment]
1+https://community.brave.com/t/unable-to-open-ads-from-the-notification-area/43094

**Comment by rivy** [comment]
This problem still exists for Windows, after this fix was merged, as of Brave version 0.63...

```
Brave is up to date
Version 0.63.20 Chromium: 73.0.3683.75 (Official Build) dev (64-bit)
```

```
C:\>ver
Microsoft Windows [Version 10.0.17134.590]
```

**Comment by tmancey** [comment]
@rivy Can you please raise a new issue, thank you
